lua/lazyvim/plugins/extras/editor/fzf.lua

```diff
@@ -208,6 +208,7 @@ return {
     keys = {
       { "<c-j>", "<c-j>", ft = "fzf", mode = "t", nowait = true },
       { "<c-k>", "<c-k>", ft = "fzf", mode = "t", nowait = true },
+      { "<c-cr>", "<cmd>FzfLua resume<cr>", desc = "Resume Picker" },
```

Comment:
That is already mapped to `<leader>sR`. I don't believe there's a need for that.

Reply:
@dpetka2001 then maybe I should use that keymap for telescope...

Reply:
I didn't actually test what your code for Telescope does and how it might be different than `Telescope resume` that is also mapped to `<leader>sR`, so maybe keep it as it is?? Wait also for maintainer's input just to be sure.